ChangeLog

```diff
@@ -184,6 +184,10 @@ Release date: TBA
 
   Closes #4324
 
+* Avoid reporting ``superfluous-parens`` on expressions using the ``is not`` operator.
+
+  Closes #5930
+
```

Comment:
Even though the top of this doc says to port "new features" to what's new/*.rst, we've been doing so for anything user-facing. So a copy-paste to 2.13.rst would be great.